services/app/src/actions/Quest.tsx

```diff
@@ -75,3 +77,15 @@ export function loadNode(node: ParserNode, details?: QuestDetails) {
     }
   };
 }
+
+export function userQuestsDelta(delta: Partial<UserQuestsType>) {
```

Comment:
Shouldn't these be in `/actions/QuestHistory`?

Reply:
True that. Fix'd.